playwright.config.js

```diff
@@ -16,6 +18,8 @@ export default defineConfig({
     baseURL: 'https://www.globalsqa.com/',
     testIdAttribute: 'id'
   },
+      timeout: 60000,  // Global timeout for all tests ***
```

Comment:
The comment on this line seems to be incomplete or incorrectly formatted. The '***' symbols are not standard for comments and might have been left unintentionally. Consider removing them or replacing them with a proper comment format if additional information is needed.